src/mockHttpModule.ts

```diff
@@ -0,0 +1,167 @@
+import {
+  captureStack, getErrorWithCorrectStack, getHeadersForHttp, getRequestMethodForHttp, getUrlForHttp, parseQueryForHttp,
+} from './utils';
+import { findMockedEndpointForHttp, findMockedMethodForHttp, getMockedEndpointMetadata } from './mockedEndpointsService';
+import { isRealNetworkAllowed } from './settings';
+import { NetmockResponseType } from './types';
+import { isInstanceOfNetmockResponse } from './NetmockResponse';
+
+function handleNotMockedResponse(request: HttpRequest, requestCallback?: CallBack, isHttpsRequest?: boolean) {
+  const originalModule = isHttpsRequest ? global.originalHttps : global.originalHttp;
+  const func = originalModule.request;
+  const url = getUrlForHttp(request);
+  const method = getRequestMethodForHttp(request);
+  const initialResponseObject = {
+    headers: {},
+    location: 'BLA',
+  };
+  if (isRealNetworkAllowed(url)) {
+    return func(request, requestCallback);
```

Comment:
using `originalModule.request` directly instead saving it to func makes it more readable